.puppet-lint.rc

```diff
@@ -1,3 +1 @@
 --relative
---no-strict_indent-check
---no-manifest_whitespace_missing_newline_end_of_file-check
```

Comment:
by removing those, you should have got lint errors. Check my pr with fixes here -> https://github.com/puppetlabs/puppetlabs-peadm/pull/381